rangeseekbar/src/main/java/org/florescu/android/rangeseekbar/RangeSeekBar.java

```diff
@@ -606,8 +606,7 @@ protected synchronized void onDraw(@NonNull Canvas canvas) {
         mRect.right = getWidth() - padding;
         canvas.drawRect(mRect, paint);
 
-        boolean selectedValuesAreDefault = (getSelectedMinValue().equals(getAbsoluteMinValue()) &&
-                getSelectedMaxValue().equals(getAbsoluteMaxValue()));
+        boolean selectedValuesAreDefault = (normalizedMinValue < 0.00001 && normalizedMaxValue > 0.99999);
```

Comment:
I think perhaps a better way of doing this is to have an acceptable "delta", below which the values are considered default. By default, this would be zero, but it can be set to something else. 
So for example...
boolean selectedValuesAreDefault = (normalizedMinValue <= minDeltaForDefault && normalizedMaxValue >= 1-minDeltaForDefault);

And separately somewhere a public method for overriding minDeltaForDefault. Does that work for your use-case?